src/pages/komodo-defi-framework/tutorials/listing-a-new-coin/index.mdx

```diff
@@ -125,13 +125,28 @@ The token's contract **must** have `approve` and `transferFrom` methods. Additio
-| ----------------------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| chain\_id                                 | integer | To find the EVM chain ID, see [Chainlist](https://chainlist.org/)                                                                                                                                                   |
-| protocol.type                             | string  | Platform / protocol - e.g `ETH` for Ethereum, `ERC20` for ERC20 tokens on the Ethereum network                                                                                                                      |
-| protocol.protocol\_data                   | object  | Required for tokens only.                                                                                                                                                                                           |
-| protocol.protocol\_data.platform          | string  | The parent coin of the token's platform - e.g `MATIC` for PLG20 tokens                                                                                                                                              |
-| protocol.protocol\_data.contract\_address | string  | **Must be mixed case** The indentifying hex string for the token's contract. Can be found on sites like [EthScan](https://etherscan.io/), [BscScan](https://bscscan.com/) & [PolygonScan](https://polygonscan.com/) |
+| Parameter                                 | Type    | Description                                                                                                                                                                                                                                                         |
+| ----------------------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| chain\_id                                 | integer | To find the EVM chain ID, see [Chainlist](https://chainlist.org/)                                                                                                                                                                                                   |
+| gas\_limit                                | object  | Optional, required for tokens only. Defines more precise gas prices for swap or transaction operations, useful for tokens called ovcer proxy contracts which often require more than the default gas. See the `Gas Limit Options` table below for more information. |
```

Comment:
> useful for tokens called ovcer proxy contracts which often require more than the default gas.

It's used for the reverse of this actually, we use default limits that allow tokens that use proxy contracts but we can use these parameters to lower limits for other tokens if needed. @dimxy please confirm this and also give this docs PR a review.

Reply:
Yes all gas_limit params are optional and internally set to default values (which maybe too high for certain tokens, so could be tuned for new or existing evm tokens)